docs/faq/wallets-and-seeds.md

```diff
@@ -70,27 +70,7 @@ A public key address, also called Pay-To-Pubkey (P2Pk), can be identified with i
 
 #### 8. What are the different types of addresses?
 
-A Decred address[^14995] is actually just a representation of a public key (which itself could be a script hash) along with a 2-byte prefix which identifies the network and type and a checksum suffix in order to detect improperly entered addresses.
```

Comment:
> a checksum suffix in order to detect improperly entered addresses.

The new page loses this detail. Can we include it please? Maybe including how many bytes the checksum consists of.

Reply:
Good catch. Was meaning to include that, but got lost in my wordsmithing:)

Have added detail to first para:

> Each address has a 2-byte prefix than can be used to identify its type, and a checksum suffix to detect improperly entered addresses. The first byte of the prefix identifies the network...

Reply:
I can also add the number of bytes the checksum uses if someone can provide. I assume it's 4 like Bitcoin addresses, but don't want to assume and can't find this in our existing docs.

Reply:
2-byte prefix, fingerprint, 4-byte checksum.  The checksum is 4 bytes of `blake256(blake256(prefix||checksum))`.